src/vs/workbench/services/issue/electron-sandbox/issueService.ts

```diff
@@ -185,6 +185,13 @@ export class NativeIssueService implements IWorkbenchIssueService {
 			githubAccessToken
 		}, dataOverrides);
 
+		if (issueReporterData.extensionId) {
+			const extensionExists = extensionData.some(extension => extension.id === issueReporterData.extensionId);
+			if (!extensionExists) {
```

Comment:
Should the function return early in this case?

Reply:
I think i didn't want to `throw` here because we have a `selected extension` option, where no extension is selected. If we throw, nothing happens, if we allow it to go thru, it goes into this state with the console error

![Screenshot 2024-02-21 at 4 09 32 PM](https://github.com/microsoft/vscode/assets/54879025/0384cf14-cd08-448f-befa-3cf8254a400f)